examples/external-security/build.gradle

```diff
@@ -0,0 +1,13 @@
+plugins {
+	// The properties plugin must be applied BEFORE ml-gradle so that ml-gradle sees the results of the properties plugin
+	// processing the gradle-(environmentName).properties files.
+	id "net.saliman.properties" version "1.4.6"
+
+	id "com.marklogic.ml-gradle" version "3.6.0"
+}
+
+task printHost {
```

Comment:
Is this task needed for the purpose of showing how to configure an external security?